_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,67 @@
+
+[@wb9688](https://github.com/wb9688) and [@B0pol](https://github.com/B0pol) (in his second contribution) added support for more subdomains. Now `music.youtube.com` and `media.ccc.de` are natively supported by NewPipe, thanks you two. In the last point to this section, [@jonas-skywalker](https://github.com/jonas-skywalker) (may the force be with you) improved the search section. Now the magnifying glass vanishes once you start typing.
+
+## Fixes a.k.a. NewPipe crashes less \o/
+
+Android 9 changed how databases behave, effectively destroying backups and exporting remote playlists - not anymore, thanks to [@yausername](https://github.com/yausername). [@XiangRongLin](https://github.com/XiangRongLin) improved sharing a downloaded video to other apps, which crashed the app before this release. The app also crashed when you tried to download a really large video, which [@XiangRongLin](https://github.com/XiangRongLin) fixed in his third contribution. The seek duration setting was not properly shown in any other language then English, but [@B0pol](https://github.com/B0pol) let them magically appear again. And finally, [@kapodamy](https://github.com/kapodamy) took care of a nasty bugs, which made some WebM videos non seekable. Now you can seek all you want.
+
+All of the following fixes were done by the incredible [@mauriciocolli](https://github.com/mauriciocolli). Overall, he brought seven (!) different contributions in todays release. He solved an issue where the last screen of the app could be popping up during the app exit animation. A bug in the main screen resulted to tabs not being restored correctly if the app was put to sleep by Android. The main screen had another issue, especially with the menu, which could vanish until the user interacted with the app again. Sharing a YouTube link to NewPipe could end up crashing NewPipe if certain characters were in those links, not an issue anymore. And finally, he improved the handling of unavailable content on YouTube, which resulted in a crash previously.
+
+## Development
+
+This release also includes two big improvements of the development process. [@Stypox](https://github.com/Stypox) and [@mauriciocolli](https://github.com/mauriciocolli) changed the way `.apk` files are generated in the development process. This makes it way easier to test a change on your device, since you can have all of these NewPipe apps alongside each other, as long as you have the storage for them.
+
+[@Poolitzer](https://github.com/poolitzer), who already took over the latest blog posts, now created up-to-date issue templates, which makes it easier for you to report a bug or suggest a feature. The final form of them that you are experiencing is the result of a combined effort of the whole community, thanks for everyone who forced me to implement and discuss their suggestions.
```

Comment:
```suggestion
[@Poolitzer](https://github.com/poolitzer) (that's me!), who already took over the latest blog posts, created up-to-date issue templates, which makes it easier for you to report a bug or suggest a feature. Their final form that you are experiencing is the result of the combined effort of the whole community, thanks to everyone who forced him to implement and discuss their suggestions.
```

Reply:
I love the thats me part, but why change the last me to him? we could just stay with me imo

Reply:
Because you're talking about yourself in third person at the beginning. If you want to make the whole thing first person, then do something like:

"I (Poolitzer) had earlier taken over making the blog posts. Now I have created up to date..."

Reply:
hmm true. Maybe I should just delete that half sentence, doesnt make much sense their.

Reply:
Alright. I'll review it after your changes.

Reply:
Tada